nbdime/webapp/nbdimeserver.py

```diff
@@ -50,9 +49,7 @@ def base_args(self):
             'closable': self.params.get('closable', False),
             'savable': fn is not None,
             'baseUrl': self.nbdime_base_url,
-            'hideUnchanged': self.params.get('hide_unchanged', True),
-            'mathjaxUrl': self.mathjax_url,
-            'mathjaxConfig': self.mathjax_config,
+            'hideUnchanged': self.params.get('hide_unchanged', True)
```

Comment:
another trailing comma (we could fix these with `black` later, but now good for diff)